packages/proto/src/lib/light_push.ts

```diff
@@ -1,9 +1,12 @@
 /* eslint-disable import/export */
```

Comment:
@fryorcraken do we have an issue tracking removal of these? Or there is another reason of having it apart from continuity of updates to the file?

Reply:
@weboko This is autogenerated code by `protons`. Not sure what you mean by "continuity of updates".

Reply:
I see, if it is auto generated then it makes sense to keep.